examples/src/main/python/ml/simple_params_example.py

```diff
@@ -18,36 +18,30 @@
 from __future__ import print_function
 
 import pprint
-import sys
 
-from pyspark import SparkContext
 from pyspark.ml.classification import LogisticRegression
 from pyspark.mllib.linalg import DenseVector
-from pyspark.mllib.regression import LabeledPoint
-from pyspark.sql import SQLContext
+from pyspark.sql import SparkSession
 
 """
-A simple example demonstrating ways to specify parameters for Estimators and Transformers.
+An example demonstrating ways to specify parameters for Estimators and Transformers.
 Run with:
   bin/spark-submit examples/src/main/python/ml/simple_params_example.py
 """
 
 if __name__ == "__main__":
-    if len(sys.argv) > 1:
-        print("Usage: simple_params_example", file=sys.stderr)
-        exit(1)
-    sc = SparkContext(appName="PythonSimpleParamsExample")
-    sqlContext = SQLContext(sc)
+    spark = SparkSession \
```

Comment:
You are right. `model1.extractParamMap()` and `model2.extractParamMap()` are always empty.
And it fails with 
```
16/05/12 09:58:48 WARN TaskSetManager: Lost task 2.0 in stage 40.0 (TID 152, localhost): java.lang.IllegalArgumentException: requirement failed: Logistic Regression getThreshold found inconsistent values for threshold (0.5) and thresholds (equivalent to 0.55)
        at scala.Predef$.require(Predef.scala:224)
        at org.apache.spark.ml.classification.LogisticRegressionParams$class.checkThresholdConsistency(LogisticRegression.scala:143)
```
I will revert this change. Thanks!